packages/shared-component--signpost/src/signpost.html

```diff
@@ -1,83 +1,91 @@
 {% macro render(content, cms, config) %}
-<div class="coop-c-signpost">
-  {% if content.fields.link and content.fields.link.sys.id %}
-  {% with entryLink = cms.get_entry(content.fields.link.sys.id) %}
+  <div class="coop-c-signpost">
+    {% if content.fields.link and content.fields.link.sys.id %}
+      {% with entryLink = cms.get_entry(content.fields.link.sys.id) %}
 
-  {% if entryLink.type == 'internalLink' %}
-  {% if entryLink.data.fields.page %}
-  {% with internalLink = cms.get_entry(entryLink.data.fields.page.sys.id) %}
-  <a href="{{tenant_url_path}}{{ entryLink.fields.url.data }}" class="coop-c-signpost__link" data-contenttype="Signpost"
-    data-contentparent="{{ config.contentParent|safe }}" data-linktext="{{ content.fields.title|safe }}">
-    {% endwith %}
-    {% endif %}
-    {% elif entryLink.fields.url.data %}
-    <a href="{{tenant_url_path}}{{ entryLink.fields.url.data }}" class="coop-c-signpost__link"
-      data-contenttype="Signpost" data-contentparent="{{ config.contentParent|safe }}"
-      data-linktext="{{ content.fields.title|safe }}">
-      {% endif %}
+        {% if entryLink.type == 'internalLink' %}
+          {% if entryLink.data.fields.page %}
+            {% with internalLink = cms.get_entry(entryLink.data.fields.page.sys.id) %}
+              <a
+                href="{{tenant_url_path}}{{ entryLink.fields.url.data }}"
```

Comment:
Change this: `{{tenant_url_path}}{{ entryLink.fields.url.data }}` 
To this :   `{{tenant_url_path}}{{ internalLink.full_url_path }}`